### DISCUSSION TITLE
图片添加说明文字

### DISCUSSION THREAD

**Discussion by oYZYo** [tuna/thuthesis#887] [Q&A] [answered]
“国外的期刊习惯将图表的标题和说明文字写成一段，需要改写为标题只含图表的名称，其他说明文字以注释方式写在图表下方，或者写在正文中。”
所以我可以把标题和说明文字放在一起写成一段吗？还是非得改写

**Comment by zepinglee** [comment, marked as answer]
按照《指南》原则上要将图题和图注分开，但是具体主要取决于导师的要求。

**Reply by oYZYo** [reply]
好的谢谢
